src/main/java/io/anserini/index/IndexCollection.java

```diff
@@ -50,6 +47,49 @@
 public final class IndexCollection {
   private static final Logger LOG = LogManager.getLogger(IndexCollection.class);
 
+  public static final class Args {
+
+    // required arguments
+
+    @Option(name = "-input", metaVar = "[Path]", required = true, usage = "collection path")
+    public String input;
+
+    @Option(name = "-index", metaVar = "[Path]", required = true, usage = "index path")
+    public String index;
+
+    @Option(name = "-threads", metaVar = "[Number]", required = true, usage = "Number of Threads")
+    public int threads;
+
+    @Option(name = "-collection", required = true, usage = "collection class in io.anserini.collection")
+    public String collectionClass;
+
+    @Option(name = "-generator", required = true, usage = "document generator in io.anserini.index.generator")
+    public String generatorClass;
+
+    // optional arguments
+
+    @Option(name = "-memorybuffer", usage = "memory buffer size")
+    public int memorybufferSize = 2048;
+
+    @Option(name = "-keepstopwords", usage = "boolean switch to keep stopwords")
+    public boolean keepstop = false;
+
+    @Option(name = "-positions", usage = "boolean switch to index positions")
+    public boolean positions = false;
+
+    @Option(name = "-docvectors", usage = "boolean switch to store document vectors")
+    public boolean docvectors = false;
+
+    @Option(name = "-storeTransformedDocs", usage = "boolean switch to store transformed document text")
```

Comment:
Yes, I think they should be consistent. Anything we add later on should follow the same style.